.vscode/settings.json

```diff
@@ -0,0 +1,8 @@
+{
+    "python.testing.pytestArgs": [
+        "tests"
+    ],
+    "python.testing.unittestEnabled": false,
+    "python.testing.nosetestsEnabled": false,
```

Comment:
For future reference we will be adding files that are specific to your coding environment such as .vscode settings into a .gitignore. This is just to prevent conflicts with other peoples' settings when they work on the same project as you.